src/lib/features/feature-lifecycle/feature-lifecycle-service.ts

```diff
@@ -155,4 +161,9 @@ export class FeatureLifecycleService extends EventEmitter {
         ]);
         this.emit(STAGE_ENTERED, { stage: 'archived' });
     }
+
+    private async featureRevived(feature: string) {
+        await this.featureLifecycleStore.delete(feature);
```

Comment:
delete all previous stages

Reply:
LG, but I was thinking about it. If we want to measure how long we were in each stage, this makes it not countable, the first stage. Unless we take it from feature created date.
But if we use created date, then after revive, the created date could have been 2 years ago.

Reply:
We need an opinion here. One option would be to delete all events expect from the initial event. So when you revive you only erase later stages.

Reply:
Since I don't know which is better I'll merge it now with an easy option to change it later